pyproject.toml

```diff
@@ -97,6 +97,7 @@ dev = [
     "vcrpy==4.4.0",
     "pyinstrument>=4.5.3",
     "scikit-optimize>=0.9.0",
+    "icecream>=2.1.3",
```

Comment:
I didn't know this package, this is nice!

Reply:
I found it on youtube lol!